lib/attr.gd

```diff
@@ -37,8 +37,14 @@
 ##
 ##  <Description>
 ##  This info class (together with <Ref InfoClass="InfoWarning"/> is used
-##  for messages about attribute storing  being  disabled  (at  level  2)  or
-##  enabled (level 3). It may be  used  in  the  future  for  other  messages
+##  for messages about attributes. The current warnings are:
+##  <List>
+##    <Item> Attribute storing  being  disabled  (level  2) </Item>
```

Comment:
I don't understand this. What is being warned about and when? I don't have the code at hand right now; perhaps the warning is issued if one uses an attribute setter on an object which does not store attributes resp. for which attribute storing is disabled? Or perhaps this is for attributes for which attribute storing is disabled (does such a thing exist? I seem to vaguely recall it, but can't right now think of an application)